### PR TITLE
doc: peering.rst, fix typo

### PR DESCRIPTION
**Pull request by badone**
Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

### CODE REVIEW

File: doc/dev/peering.rst [hunk 1]
```diff
@@ -145,7 +145,7 @@ history of past maps to generate a set of *past intervals* to
 determine which OSDs must be consulted before we can successfully
 *peer*.  The set of past intervals is bounded by *last epoch started*,
 the most recent *past interval* for which we know *peering* completed.
-The process by with an OSD discovers a PG exists in the first place is
+The process by wich an OSD discovers a PG exists in the first place is
```
Comment (gaowanlong) [inline doc/dev/peering.rst]: Introduced another typo...

**Comment by badone** [comment]
Oh wow, thanks. Fixing it now.

**Comment by badone** [comment]
retest this please.

**Comment by badone** [comment]
retest this please

**Comment by badone** [comment]
retest this please

**Comment by badone** [comment]
retest this please

**Comment by badone** [comment]
retest this please

**Comment by badone** [comment]
retest this please

**Comment by badone** [comment]
retest this please

**Pull request merged**